patches/minecraft/net/minecraft/pathfinding/ClimberPathNavigator.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/pathfinding/ClimberPathNavigator.java
++++ b/net/minecraft/pathfinding/ClimberPathNavigator.java
+@@ -38,7 +38,7 @@
+          super.func_75501_e();
+       } else {
+          if (this.field_179696_f != null) {
+-            if (!this.field_179696_f.func_218137_a(this.field_75515_a.func_213303_ch(), (double)this.field_75515_a.func_213311_cf()) && (!(this.field_75515_a.func_226278_cu_() > (double)this.field_179696_f.func_177956_o()) || !(new BlockPos((double)this.field_179696_f.func_177958_n(), this.field_75515_a.func_226278_cu_(), (double)this.field_179696_f.func_177952_p())).func_218137_a(this.field_75515_a.func_213303_ch(), (double)this.field_75515_a.func_213311_cf()))) {
++            if (!this.field_179696_f.func_218137_a(this.field_75515_a.func_213303_ch(), Math.max((double)this.field_75515_a.func_213311_cf(), 1.0D)) && (!(this.field_75515_a.func_226278_cu_() > (double)this.field_179696_f.func_177956_o()) || !(new BlockPos((double)this.field_179696_f.func_177958_n(), this.field_75515_a.func_226278_cu_(), (double)this.field_179696_f.func_177952_p())).func_218137_a(this.field_75515_a.func_213303_ch(), Math.max((double)this.field_75515_a.func_213311_cf(), 1.0D)))) {
```

Comment:
A proper fix would be nicer then just enforing a min value of 1.0, but I guess that would be a too complicated patch?
Also, please add a comment saying `FORGE: Fix MC-94054 Entities using ClimberPathNavigator spinning in navigation when width is small` or something like that.

Reply:
Enforcing the min value didn't really change anything in the pathfinding other than fix the spinning from what I could see. But yes I also think fixing it the proper way would be more complicated.